api/api_test.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/planetdecred/dcrlibwallet/utils"
+)
+
+func TestGetBestBlock(t *testing.T) {
+	chainParams, err := utils.ChainParams("mainnet")
+	if err != nil {
+		log.Error("Error creating chain params.")
+		return
+	}
+
+	service := NewService(chainParams)
+	height := service.GetBestBlock()
```

Comment:
seems like this test relies or "3rd party API" working while this test is executing, this will likely result in some intermittent failures of this test, and it could be annoying to go look at it every time it fails in false-positive manner (on CI especially),

I don't know what's the approach `dcrlibwallet` is following on this matter, just thought worth mentioning